### PR TITLE
Remove jsafrane from some tests

### PR DESCRIPTION
**Pull request by jsafrane**
I do not know anything about ESIPP nor clouddns and I have never touched these tests. It would be better to assign flakes to someone else.

@bprashanth @quinton-hoole, PTAL. I took your names as authors of these two tests.

### CODE REVIEW

File: test/test_owners.csv [hunk 1]
```diff
@@ -520,7 +520,7 @@ k8s.io/kubernetes/federation/apis/federation/install,nikhiljindal,0
 k8s.io/kubernetes/federation/apis/federation/validation,nikhiljindal,0
 k8s.io/kubernetes/federation/pkg/dnsprovider,sttts,1
 k8s.io/kubernetes/federation/pkg/dnsprovider/providers/aws/route53,cjcullen,1
-k8s.io/kubernetes/federation/pkg/dnsprovider/providers/google/clouddns,jsafrane,1
+k8s.io/kubernetes/federation/pkg/dnsprovider/providers/google/clouddns,quinton,1
```
Comment (ixdy) [inline test/test_owners.csv]: maybe assign this to @madhusudancs instead.

**Comment by jsafrane** [comment]
rebased and used @madhusudancs instead of Quinton. Sorry it took so long.

**Comment by madhusudancs** [comment]
Alright, I am signing up for it. LGTM.

Wait for @bprashanth too.

**Comment by deads2k** [comment]
@k8s-bot kops aws e2e test this

**Comment by deads2k** [comment]
> Alright, I am signing up for it. LGTM.
> 
> Wait for @bprashanth too.

he gave a conditional lgtm up here https://github.com/kubernetes/kubernetes/pull/38107#discussion_r91187054

**Pull request merged**